app.js

```diff
@@ -41,6 +41,12 @@ app.get('/crit/:crit_id', (req, res) => {
 	});
 	
 });
+//welcome page route
+app.get('/Welcome', (req, res) => {
+	res.render('NEWUSER');})

```

Comment:
Move the curlybracket and paren down to the next line.

Also, add a semi-colon after the paren.